trustmanager/x509utils.go

```diff
@@ -440,6 +451,11 @@ func EncryptPrivateKey(key data.PrivateKey, passphrase string) ([]byte, error) {
 		return nil, err
 	}
 
+	if encryptedPEMBlock.Headers == nil {
```

Comment:
@diogomonica fixed in https://github.com/docker/notary/commit/0465365fb6fb9d1f5f995e0f19f772f5334fbe09